addons/mail/views/res_users_views.xml

```diff
@@ -29,5 +29,17 @@
                 </data>
             </field>
         </record>
+
+        <!-- Blocked notification alert -->
+        <record id="view_users_form_inherit_mail_notification_alert" model="ir.ui.view">
+            <field name="name">res.users.preferences.form.notification.alert</field>
+            <field name="model">res.users</field>
+            <field name="inherit_id" ref="base.view_users_form_simple_modif"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='image_1920']" position="before">
+                    <widget name="notification_alert"/>
+                </xpath>
+            </field>
+        </record>
```

Comment:
> > Also `view_users_form_mail` is already inherited in mail, maybe it can be combined as to not have a new template in the first place?
> 
> The field Odoobot_state is part of mail_bot. It's in @alexkuhn review.

I think @seb-odoo meant `base.view_users_form_simple_modif` is already inherited in mail by `view_users_form_simple_modif_mail`, so could just add `xpath` on this latter inherit?

Reply:
OK ! got it done :)